app/models/ems_folder.rb

```diff
@@ -19,6 +19,13 @@ class EmsFolder < ApplicationRecord
   virtual_has_many :vms,               :uses => :all_relationships
   virtual_has_many :miq_templates,     :uses => :all_relationships
   virtual_has_many :hosts,             :uses => :all_relationships
+  virtual_column   :total_configured_systems, :type => :integer
+
+  #
+  # Methods for virtual columns
+  #
+
+  delegate :total_configured_systems, :to => :ext_management_system
```

Comment:
Based on the decsription, I believe you are here, correct?
![image](https://cloud.githubusercontent.com/assets/651659/16620688/6694f2ee-4361-11e6-8c5b-90857a0e7c69.png)

I see the same error when trying to sort by Total Configured Systems.

However, delegating `total_configured_systems` to `ext_management_system` will get you the total number of Configured System for the entire provider which would not be the correct number.  Instead, you want the total for the Inventory Group.  I think your method should be:
```ruby
def total_configured_systems
  configured_systems.count
end
```

This also makes me question how the table was build the first time since this method doesn't exist...  There is probably another location that would benefit from using this method.

Also, I believe this logic should live in the InventoryGroup model, since it only applies there.